CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
 cmake_minimum_required(VERSION 2.8.12.2 FATAL_ERROR)
+
 if(POLICY CMP0063)
   cmake_policy(SET CMP0063 NEW)
 endif()
 
 project(prometheus-cpp)
-enable_testing()
+
 include(GNUInstallDirs)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/cmake)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-if(NOT DEFINED CMAKE_CXX_STANDARD AND UNIX)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
That only works for CMake ~3.4.

Reply:
It is available with 3.1 :) (https://cmake.org/cmake/help/v3.1/variable/CMAKE_CXX_STANDARD.html?highlight=cmake_cxx_standard). Why was 3.0 chosen? Is it possible to raise the version? (most Linux distributions ship with 3.5 or higher and on other platforms which do not have a package manager the user has to install CMake anyway manually)

Reply:
It might make sense to raise the limit to 3.5. Both Ubuntu trusty and RHEL 6 EPEL provide a CMake 3.5 (or later package)